src/shared/components/MonacoEditorESM/autocompletion/useAutocompleteWorker.js

```diff
@@ -42,11 +43,12 @@ export function useAutocompleteWorker({
     autocompletionDisabled = true;
   }
 
-  const { schema, loading, error } = useGetSchema({
+  const { fetchedSchema, loading, error } = useGetSchema({
```

Comment:
Maybe make like this to have it more clear
```suggestion
  const { schemas: fetchedSchema, loading, error } = useGetSchema({
```